cdap-docs/reference-manual/source/http-restful-api/http-restful-api-v3/lifecycle.rst

```diff
@@ -552,6 +552,24 @@ and can limit the number of returned records.
 The *runid* field is a UUID that uniquely identifies a run within CDAP,
 with the start and end times in seconds since the start of the Epoch (midnight 1/1/1970).
 
+To fetch run record for a particular run for a program, use::
+
+  GET <base-url>/namespaces/<namespace>/apps/<app-id>/<program-type>/<program-id>/runs/<run-id>
+
```

Comment:
Please insert a table with "Parameter - Description" for the four parameters (<app-id>, <program-type>, <program-id>, <run-id>)